### PR TITLE
VideoPress: TypeScriptify code

### PR DESCRIPTION
**Pull request by retrofox**
Fixes #

#### Changes proposed in this Pull Request:

* VideoPress: TypeScriptify code. Janitorial tasks.

#### Testing instructions:

* Everything should work as expected.
* So far, you should check "[Privacy](https://github.com/Automattic/jetpack/pull/27401) and [Rating](https://github.com/Automattic/jetpack/pull/27415)" panel.

### CODE REVIEW

File: projects/packages/videopress/src/client/block-editor/blocks/video/types.ts [hunk 1]
```diff
@@ -1,14 +1,27 @@
...
+
+export type videoBlockColorAttributesProps = {
+	seekbarPlayedColor?: string;
+	seekbarLoadingColor?: string;
+	seekbarColor?: string;
 };
 
-export type VideoBlockAttributes = {
-	id?: number;
+export type videoBlockAttributes = videoBlockColorAttributesProps & {
```
Comment (dhasilva) [inline projects/packages/videopress/src/client/block-editor/blocks/video/types.ts]: I think the convention we have been using is to use `PascalCase` for types and interfaces and `camelCase` for their properties.

Comment (retrofox) [reply projects/packages/videopress/src/client/block-editor/blocks/video/types.ts]: ohhh :'(

Comment (retrofox) [reply projects/packages/videopress/src/client/block-editor/blocks/video/types.ts]: https://github.com/Automattic/jetpack/pull/27419/commits/e8d2275c5bf857b1957147980a5862753aef4876

**Review by dhasilva — APPROVED** [review]
:rocket:

**Pull request merged**